app/views/shared/search/_keyword.html.slim

```diff
@@ -1,4 +1,5 @@
 = f.govuk_text_field :keyword,
   label: { text: t("jobs.filters.keyword"), size: "s" },
-  placeholder: t("jobs.filters.keyword_hint"),
+  hint: { text: (@nqt_job_alert == true ? t("jobs.filters.keyword_nqt_hint") : nil) },
```

Comment:
You don't need `== true`, just `@nqt_job_alert ? ___ : ___` is enough to test if it's truthy 💡